integration-tests/src/test/java/gov/uk/address/api/stepdefinitions/AddressSteps.java

```diff
@@ -1,53 +1,62 @@
+import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.nimbusds.jwt.SignedJWT;
 import gov.uk.address.api.client.AddressApiClient;
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
-import software.amazon.awssdk.services.sqs.model.Message;
-import uk.gov.di.ipv.cri.common.library.aws.CloudFormationHelper;
-import uk.gov.di.ipv.cri.common.library.aws.SQSHelper;
 import uk.gov.di.ipv.cri.common.library.client.ClientConfigurationService;
+import uk.gov.di.ipv.cri.common.library.domain.AuditEvent;
+import uk.gov.di.ipv.cri.common.library.domain.TestHarnessResponse;
 import uk.gov.di.ipv.cri.common.library.stepdefinitions.CriTestContext;
+import uk.gov.di.ipv.cri.common.library.util.JsonSchemaValidator;
 
 import java.io.IOException;
+import java.net.URISyntaxException;
 import java.net.URLEncoder;
 import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.text.ParseException;
-import java.util.Collections;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 
-import static java.util.Map.entry;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
 
 public class AddressSteps {
+    private static final String ADDRESS_START_SCHEMA_FILE = "/schema/IPV_ADDRESS_CRI_START.json";
     private final ObjectMapper objectMapper;
     private final AddressApiClient addressApiClient;
     private final CriTestContext testContext;
-    private final SQSHelper sqs;
+    private final String addressStartJsonSchema;
 
     private String postcode;
     private String uprn;
     private String countryCode;
 
-    private final String auditEventQueueUrl =
-            CloudFormationHelper.getOutput(
-                    CloudFormationHelper.getParameter(System.getenv("STACK_NAME"), "TxmaStackName"),
-                    "AuditEventQueueUrl");
-
     public AddressSteps(
-            ClientConfigurationService clientConfigurationService, CriTestContext testContext) {
+            ClientConfigurationService clientConfigurationService, CriTestContext testContext)
+            throws IOException, URISyntaxException {
         this.objectMapper = new ObjectMapper();
         this.addressApiClient = new AddressApiClient(clientConfigurationService);
         this.testContext = testContext;
-        this.sqs = new SQSHelper(null, this.objectMapper);
+
+        Path schemaPath =
+                Paths.get(
+                        Objects.requireNonNull(
```

Comment:
I'm not sure this NonNull check really does anything beneficial?

Reply:
when just adding `AddressSteps.class.getResource(ADDRESS_START_SCHEMA_FILE).toURI()` receives a warning that "Method invocation 'toURI' may produce 'NullPointerException'" so it suggest to add `Objects.requireNonNull` to the method.

Reply:
Ok leave it as is then with it being test code it's ok just just throw a NullPointer